sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java

```diff
@@ -48,48 +51,72 @@ public void afterClass() {
     }
 
     @Test(groups = {"simple"}, timeOut = TIMEOUT)
-    public void createItem_withBulk() {
-        int totalRequest = getTotalRequest();
+    public void createItem_withBulk() throws InterruptedException {
+        int totalRequest = getTotalRequest(180, 200);
+
+        PartitionKeyDefinition pkDefinition = new PartitionKeyDefinition();
+        pkDefinition.setPaths(Collections.singletonList("/mypk"));
+        CosmosAsyncContainer bulkAsyncContainerWithThroughputControl = createCollection(
+            this.bulkClient,
+            bulkAsyncContainer.getDatabase().getId(),
+            new CosmosContainerProperties(UUID.randomUUID().toString(), pkDefinition));
+
+        ThroughputControlGroupConfig groupConfig = new ThroughputControlGroupConfigBuilder()
```

Comment:
I think this should be a different test case vs without throughput enabled.

Reply:
Agreed

Reply:
Fixed